src/QueryRenderer.ts

```diff
@@ -186,15 +186,18 @@ class QueryRenderChild extends MarkdownRenderChild {
             const footnotes = listItem.querySelectorAll('[data-footnote-id]');
             footnotes.forEach((footnote) => footnote.remove());
 
-            const postInfo = listItem.createSpan();
             const shortMode = this.query.layoutOptions.shortMode;
 
+            if (this.query.layoutOptions.showUrgency) {
+                this.addUrgency(listItem, task);
+            }
+
             if (!this.query.layoutOptions.hideBacklinks) {
-                this.addBacklinks(postInfo, task, shortMode, isFilenameUnique);
+                this.addBacklinks(listItem, task, shortMode, isFilenameUnique);
```

Comment:
Let's talk about the change from `postInfo` to `listItem` when we pair on urgency customisation next.

I've some questions about the consequences for users - and they will be much easier to talk about in person than via keyboard!

Reply:
OK

Reply:
Summary: Before this PR, the addBacklinks() method added a 'tasks-backlink' to the element passed in to it.

Now it creates its own nested span and styles that. This simplifies the code...